### PR TITLE
feat: add tenant invitations and tenant members resources

### PR DESCRIPTION
**Pull request by koalaty-code**
## Description

- Adds new models and interface methods for Tenant Invitations and Tenant Members

## Testing required outside of automated testing?

- [X] Not Applicable

### Screenshots (if appropriate):

- [X] Not Applicable

## Rollback / Rollforward Procedure

- [X] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: src/models/tenants.ts [hunk 1]
```diff
@@ -23,10 +23,36 @@ interface TokenTypeMetrics {
...
+  status: string;
+  expiresAt: string;
+}
+
+type CreateTenantInvitation = Pick<TenantInvitation, 'email'>;
+
+interface TenantMember extends Auditable {
+  id: string;
+  tenantId: string;
+  user: {
```
Comment (lcschy) [inline src/models/tenants.ts:39]: Can we create type for this? Could be TenantUser or something like it.

**Review by lcschy — CHANGES_REQUESTED** [review]
Just a comment

File: src/models/tenants.ts (hunk 1 above)
Comment (djejaquino) [reply src/models/tenants.ts:39]: This looks pretty similar to Auth0's `User` and our own Portal `User`. Consolidating that type in this project, along with property parsing utils seems like a good idea.

**Pull request merged**